packages/analytics-types/src/config.ts

```diff
@@ -73,6 +75,10 @@ export interface TrackingOptions {
   versionName?: boolean;
 }
 
+export interface ReactNativeTrackingOptions extends TrackingOptions {
+  adid?: boolean;
```

Comment:
feels we also need idfa/idfv for iOS?
But that's out of scope for this pr.

Reply:
Thanks for reviewing! Yes, I don't find the logic in iOS code here to generate idfa/idfv, probably this is missing. Think we can check with Brian next week, probably can be updated with the Kotlin/iOS import change (instead of duplicating some logics here).